lib/jellyfish/room_service.ex

```diff
@@ -41,9 +40,11 @@ defmodule Jellyfish.RoomService do
   @impl true
   def handle_call({:create_room, max_peers}, _from, state)
       when is_nil(max_peers) or is_number(max_peers) do
-    {:ok, room_pid} = GenServer.start_link(Room, max_peers)
+    {:ok, room_pid} = Room.start_link(max_peers)
```

Comment:
If I understand correctly, if the room dies, the entire room service goes down?

Reply:
You're absolutely correct, we also don't do kill Room GenServer and associated Socket processes on `RoomService.delete_room`, but that will be taken care of in another PR.

Reply:
I was actually referring to crashes. Normal exits will not activate the link. But I'm fine with handling them in another pull request

Reply:
Yea, I mean that in general handling processes in some more proper way is not implemented.